packages/components/src/slot-fill/fill.js

```diff
@@ -15,64 +15,56 @@ import { Consumer } from './context';
 
 let occurrences = 0;
 
-class FillComponent extends Component {
-	constructor() {
-		super( ...arguments );
-		this.occurrence = ++occurrences;
-	}
+function FillComponent( { name, getSlot, children, registerFill, unregisterFill } ) {
+	// Random state used to rerender the component if needed, ideally we don't need this
+	const [ , updateRerenderState ] = useState( {} );
+	const rerender = () => updateRerenderState( {} );
```

Comment:
According to @gaearon

https://twitter.com/dan_abramov/status/1118301474876948480

you can do this:

```js
const [, forceUpdate] = useReducer(x => x + 1, 0)
```

Reply:
yes, not sure which one is best. I guess it doesn't matter that much